### PR TITLE
feat: implemented user table design

### PR DESCRIPTION
**Pull request by Ifenna-Monanu**

### CODE REVIEW

File: src/pages/ManageUser/EachUser.tsx [hunk 1]
```diff
@@ -0,0 +1,67 @@
+import React from 'react'
+import MainContainer from '../../components/layout/MainContainer'
+import { ReactComponent as GoBack } from "../../assets/icons/goBack.svg";
```
Comment (Dverybest) [inline src/pages/ManageUser/EachUser.tsx:3]: why not create an index file in assets and export all images from there?

File: src/pages/ManageUser/transactionHistory.tsx [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+        details: "770 TF ALESSA AB...",
+        amount: "₦9,430.15",
+        date: "14:37 | 10 AUG, 2023"
+    },
+    
+]
+
+const EachTransaction = ({type, text, details, amount, date}:ITransact ) => {
+    return(
+        <div className='py-4 px-4 flex items-center justify-between border-b last:border-none border-[#E3E3E3]'>
```
Comment (Dverybest) [inline src/pages/ManageUser/transactionHistory.tsx:48]: Move this to a sperate file and consider renaming it to Transaction rather than EachTransaction

**Review by Dverybest — CHANGES_REQUESTED** [review]

File: src/pages/ManageUser/EachUser.tsx (hunk 1 above)
Comment (Ifenna-Monanu) [reply src/pages/ManageUser/EachUser.tsx]: Done

File: src/assets/icons/bell.tsx [hunk 3]
```diff
@@ -1,3 +1,13 @@
+import { DetailedHTMLProps, FC, SVGProps } from 'react';
+
+
+export const BellIcon: FC<
+  DetailedHTMLProps<SVGProps<SVGSVGElement>, SVGSVGElement>
+> = () => (
+
```
Comment (Dverybest) [inline src/assets/icons/bell.tsx:7]: why are all these svg files tsx???

Comment (Ifenna-Monanu) [reply src/assets/icons/bell.tsx:7]: I wanted to do it like we did in nine-mobile

**Review by Dverybest — APPROVED** [review]
LGTM

**Pull request merged**